docker/sparkswapd/Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /home/app
 RUN git clone -b ${BROKER_VERSION} https://github.com/sparkswap/broker.git .
 
 RUN npm install --quiet --production
-RUN npm run build -- --no-docker
+RUN npm run build -- --no-docker --no-identity --no-certs
```

Comment:
[nit] might be worth adding a comment above as to why we pass these flags when building the image